spark/src/main/scala/org/apache/comet/ExtendedExplainInfo.scala

```diff
@@ -137,18 +161,37 @@ class ExtendedExplainInfo extends ExtendedExplainGenerator {
             depth + 1,
             lastChildren :+ false,
             indent,
-            outString)
+            outString,
+            planStats)
         case _ =>
       }
       node.children.last match {
         case c @ (_: TreeNode[_]) =>
-          generateTreeString(getActualPlan(c), depth + 1, lastChildren :+ true, indent, outString)
+          generateTreeString(
+            getActualPlan(c),
+            depth + 1,
+            lastChildren :+ true,
+            indent,
+            outString,
+            planStats)
         case _ =>
       }
     }
   }
 }
 
+class PlanStats {
+  var sparkOperators: Int = 0
+  var cometOperators: Int = 0
+  var wrappers: Int = 0
+  var transitions: Int = 0
+
+  override def toString: String = {
+    s"sparkOperators=$sparkOperators, cometOperators=$cometOperators, " +
```

Comment:
Perhaps we could use a more verbose string here so that the meaning of these stats is a little more obvious